swupd verify --fix --picky deletes current kernel from /usr/lib/kernel if a newer one is installed

My system auto-updated to 18280, including kernel 4.13.5-416. I had not yet rebooted, and was still running 4.13.4-408. When I ran swupd verify --fix --picky, it deleted my currently-running kernel and related files from /usr/lib/kernel, resulting in an error:
```
REMOVING /usr/lib/kernel/org.clearlinux.native.4.13.4-408
REMOVING /usr/lib/kernel/config-4.13.4-408.native
REMOVING /usr/lib/kernel/cmdline-4.13.4-408.native
Inspected 237731 files
  0 files were missing
  0 files found which should be deleted
Calling post-update helper scripts.
[ERROR] cbm (src/bootman/update.c:L270): Cannot dermine the currently running kernel
Possible filedescriptor leak : 7 (socket:[276406])
Fix successful
```

--- a/src/extra_files.c
+++ b/src/extra_files.c
@@ -57,6 +57,7 @@ static struct fileskip {
 	int len;
 } skip_dirs[] =
     { { shortname : "/lib/modules" },
+      { shortname : "/lib/kernel" },
       { shortname : "/local" } };
 static int path_prefix_len;
 

